### ISSUE TITLE
Implement toggle Welsh accessibility feedback.

### ISSUE THREAD

**Issue by simoneduca** [dwp/design-system#227]
- [x] On the “Welsh (Cymraeg)” link on the fall-back pattern, you have Cymraeg which is Welsh, but it is missing the lang attribute. It is also mixing English and Welsh in the same anchor so you can’t just apply it directly to the `<a>` tag. In this case it would be best to use an idiomatic text element `<i>` tag. You could use a `<span>` but there is a bug in Voiceover on iOS and Mac OS X which makes it just flat out ignore the language attribute on a Span for some reason. But it works with the idiomatic text element, and it’s still valid HTML as it is a valid use of the `<i> `attribute as you’re literally changing the voice. So, for example:
 
This service is also available in `<a class="govuk-link" href="#">Welsh (<i lang="cy">Cymraeg</i>)</a>`
 

- [x] The next issue is the same, but it’s inside the first HTML code block example. The code example looks like the following:
```
<li class="dwp-language-toggle__list-item">
  <span class="govuk-visually-hidden">Newid yr iaith ir Gymraeg</span>
  <a href="?lang=cy" lang="cy" class="govuk-link">Cymraeg</a>
</li>
```
In this instance it would make sense to move the `lang=”cy”` attribute onto the `<li>` tag, because the parent will pass the `lang` attribute down for the nested child elements to inherit. So, anything inside the `<li>` would then be presumed Welsh, but anything outside that `<li>` would still be inheriting from the `lang=”en”` attribute attached to the `<html>` tag at the top of the DOM.
 

- [x] (Content) This brings us onto the next and final point which is the sentence in the accessibility requirements section which reads: “The word ‘English’ should have the HTML language attribute `en` and the word ‘Cymraeg’ should have the HTML language attribute `cy`. This could be misinterpreted. The English link would not need the attribute `lang=”en”` on the English version of the site, because it will inherit it from the `<html lang=”en”>` element at the top of the DOM. But it would need the `lang=”en”` attribute on the Welsh version of the page because it is not the primary language. Likewise, the Cymraeg link would not need the attribute `lang=”cy”` on the Welsh version of the site because it would inherit it from the `<html lang=”cy”>` element at the top of the DOM. For example: 

English mode:
```
<html lang="en"> // Sets default language as English for the whole page
<a href="?lang=en">English</a> // This inherits English from the HTML tag
<a href="?lang=cy" lang="cy">Cymraeg</a> // This needs to be specifically tagged as Welsh
```
Welsh mode:
```
<html lang="cy"> // Sets default language as Welsh for the whole page
<a href="?lang=en" lang="en">English</a> // This needs to be specifically tagged as English
<a href="?lang=cy">Cymraeg</a> // This inherits Welsh from the HTML tag
```
I think the explanation swapping the default language of the page on the HTML tag might also be valuable for teams, because the language attribute on the links will meet 3.1.2 Language of parts, but if you don’t swap the <html lang=”en”> out for <html lang=”cy”> on the Welsh version of the site you will then fail 3.1.1 Language of page.
